libc/misc/eedef.h

```diff
@@ -63,9 +63,42 @@
 #    define CCP_SPM_gc   (0x9D)
 #  endif
 #  ifndef NVMCTRL_CMD_PAGEERASEWRITE_gc
-#    define NVMCTRL_CMD_PAGEERASEWRITE_gc 3
-#  endif
-#endif
+#    if NVMCTRL_CMD_gm == 0x7F
+#      if defined (__AVR_AVR16EA28__) || defined (__AVR_AVR16EA32__) || defined (__AVR_AVR16EA48__) || \
+          defined (__AVR_AVR16EB14__) || defined (__AVR_AVR16EB20__) || defined (__AVR_AVR16EB28__) || \
+          defined (__AVR_AVR16EB32__) || defined (__AVR_AVR32EA28__) || defined (__AVR_AVR32EA32__) || \
+          defined (__AVR_AVR32EA48__) || defined (__AVR_AVR64EA28__) || defined (__AVR_AVR64EA32__) || \
+          defined (__AVR_AVR64EA48__)
```

Comment:
Just one nit: IIRC the coding rules are to break long lines before operators `||` , not after. (And to left-align the lines at the same column.)

Reply:
Of course I can change the position of the line break, but I haven't found any example in the repository where the line is break before ||. (reverse example: asmdemo.c:96). Do we have any document on code style?
And I have a question because I don't understand the indentation/left-align hit. What do you mean in this sentence 'And to left-align the lines at the same column.'? Can you show some example?

Reply:
> indentation/left-align

The alignment would be
```c++
expression A ...
|| expression B ...
```
an not
```c++
   expression A ...
|| expression B ...
```